lib/rubygems/sigstore/fulcio_api.rb

```diff
@@ -0,0 +1,31 @@
+require "faraday_middleware"
+require "openssl"
+
+class Gem::Sigstore::FulcioApi
+  def initialize(host:, token:)
+    @host = host
+    @token = token.to_s
+  end
+
+  def create(proof, pub_key)
+    connection.post("/api/v1/signingCert", {
+      publicKey: {
+        content: Base64.encode64(pub_key),
+        algorithm: "ecdsa"
```

Comment:
I think it's rsa now.

Reply:
Nevermind. Fulcio's api [only provides ecdsa](https://github.com/sigstore/fulcio/blob/main/openapi.yaml#L75).  I wonder if another part of our code is wrong, then? I don't really understand what the aglo parameter does here.

Reply:
Maybe it's the algo used to sign the signing certificate using the root Fulcio cert.  If that's the case, then once we start retrieving the root cert via the signing cert's AIA field, we'll need to verify the signature using ecdsa as well.